src/components/Application/Application.css

```diff
@@ -0,0 +1,8 @@
+.jumbotron {
+  display: flex !important;
+  justify-content: space-evenly !important;
+}
+
+.jumbotron img {
+  margin-right: 3rem;
```

Comment:
why are we using rem here? do we plan on using rem throughout the entire app instead of using pixels?